app/models/maintenance_stats/stats/bitbucket/issue_stats.rb

```diff
@@ -0,0 +1,81 @@
+module MaintenanceStats
+    module Stats
+        module Bitbucket
+            class IssueRates < MaintenanceStats::Stats::BaseStat
+                def count_issues(issues, state: nil)
+                    if state.nil?
+                      issues.size
+                    else
+                      issues.where(state: state).size
+                    end
+                  end
+            
+                  def closed_issues_count
+                    return @closed_issues_count unless @closed_issues_count.nil?
+            
+                    @closed_issues_count = count_issues(issues, state: "closed")
+                    @closed_issues_count
+                  end
+            
+                  def open_issues_count
+                    return @open_issues_count unless @open_issues_count.nil?
+            
+                    @open_issues_count = count_issues(issues, state: "open")
+                    @open_issues_count
+                  end
+            
+                  def issue_closure_rate
+                    return 1.0 if count_issues(issues) == 0
+                    closed_issues_count.to_f / count_issues(issues).to_f
```

Comment:
Should we be using `BigDecimal` here for more precision, or is floating point OK?

Reply:
I think floating should be fine, precision isn't a big factor here.